### ISSUE TITLE
WelcomeMessage not displaying

### ISSUE THREAD

**Issue by yannvr** [amwmedia/plop#140]
Using version 2.1.0
Given the plopfile:
```
module.exports = function(plop) {
  plop.setWelcomeMessage('this is welcome message')
}
```
The welcome message will not display. This happens to me as well when I have a generator.
It's strange because the plop example seems to work but I've never managed to get it working on my own plopfiles :(

**Comment by amwmedia** [comment]
If you have only 1 generator, the welcome message is skipped and you are sent right into the only generator available. Sounds like this might be what you're seeing?

**Comment by amwmedia** [comment]
I agree. I think the welcome message should show if it's specified. This was more of a consequence than a conscious decision. It came out of a request to [automatically launch the generator if there is only one available](https://github.com/amwmedia/plop/issues/67).

However, if someone added their own welcome message... it should be shown regardless of how many generators there are.
